applications/browser/package.json

```diff
@@ -72,7 +72,6 @@
     "@theia/process": "1.40.0",
     "@theia/property-view": "1.40.0",
     "@theia/scm": "1.40.0",
-    "@theia/scm-extra": "1.40.0",
```

Comment:
@danepowell could you remove the extension from the other application? It's in `applications/electron/package.json`

Reply:
Done!